test/index.js

```diff
@@ -688,21 +594,14 @@ describe('Schwifty', () => {
 
     describe('schwifty-migrate-diff require helper', () => {
 
-        it('returns null if schwifty-migrate-diff is not found', (flags) => {
+        it('returns undefined if schwifty-migrate-diff is not found', () => {
```

Comment:
Is it better to not return a controlled value? `null` Is returning null worse for a specific reason?

Reply:
`undefined` just felt more appropriate and is consistent with what we do in other repos, but I don't think one is obviously better than the other.  Really not a big deal either way!  Mostly did it out of compulsion.  When this function is used in the migrate-diff command it just looks for a falsey value anyway :P

Reply:
Just asking for learning =) the smallest peas